src/Tribe/Integrations/WPML/WPML.php

```diff
@@ -80,6 +84,40 @@ protected function hook_filters() {
 			$category_translation = Tribe__Events__Integrations__WPML__Category_Translation::instance();
 			add_filter( 'tribe_events_category_slug', array( $category_translation, 'filter_tribe_events_category_slug' ), 20, 2 );
 		}
+
+		//// WORK IN PROGRESS!!!
+
+		add_action( "tribe_repository_events_pre_get_posts", function () {
+			$request_uri = isset( $_SERVER['REQUEST_URI'] )
+				? filter_var( $_SERVER['REQUEST_URI'], FILTER_SANITIZE_STRING )
+				: false;
+			if(false === $request_uri){
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: No space after opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: No space after opening parenthesis is prohibited
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: No space before closing parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after closing parenthesis; found 0
* :no_entry_sign: No space before closing parenthesis is prohibited
* :no_entry_sign: You must have a space after a closing parenthesis
* :no_entry_sign: Space between opening control structure and closing parenthesis is required

<sub>hash: 85dc6decd4d29ed43fec735bbcb9b2d1</sub>